portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,33 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <meta charset="utf-8">
+    <script src="https://use.fontawesome.com/d1341f9b7a.js"></script>
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
+    <title>My Portfolio</title>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
+<div class="box">
+  <img src="images/beach2.0.png">
+  <h1>Welcome to My Portfolio</h1>
+    <h5>There's no formula to Life!!</h5>
+    <p>
+    Hey there, nice to meet you!!!. Swipe right to learn a little more about me. 
+    </p>
+    <ul>
+    <li><a href="https://www.instagram.com/kiba.jpeg/"><i class="fa fa-instagram" aria-hidden="true"></i></a></li>
+    <li><a href="https://github.com/kibazohb/"><i class="fa fa-github" aria-hidden="true"></i></a></li>
+    <li><a href="https://www.linkedin.com/in/briankibazohi1997/"><i class="fa fa-linkedin" aria-hidden="true"></i></a></li>
+    </ul>
+</div>
+
+<div class="content">
+    <h2> Into the Software Universe </h2>
+    <p> I still remember the first time I saw lines of code. My Dad opened up some documentation on how to develop a video game.
```

Comment:
The formatting here is quite nice, but please consider wrapping to 80 columns

Reply:
okay, another formatting issue. I'll take notes for sure.